src/material/datepicker/datepicker-toggle.scss

```diff
@@ -1,5 +1,11 @@
 @use '@angular/cdk';
 
+// We support the case where the form field is disabled, but the datepicker is not.
```

Comment:
typo: "datepicker does not"

Reply:
I think that "is not" is correct since the first part of the sentence says "the form field is disabled".